docs/cookbook-recipes/isle-v140-git-cleanup.md

```diff
@@ -0,0 +1,163 @@
+As a result of the ISLE Phase II Sprint F - Optimize Docker Images in Fall 2019, the ISLE repository has had several changes:
+
+* Original ISLE git repo size was ~182 MB and would take a minimum of 10 -15 seconds or more to clone (_depending on one's Internet connection_)
+
+- The cleaned up ISLE git repo size is now 6.6 MB and should take a minimum of less than 3 seconds to clone (_depending on one's Internet connection_)
+
+* a clean up script along with this documentation was created and run on the ISLE repository for the ISLE v.1.4.0 release.
+
+- Any new pull request will have to be created from a forked or cloned ISLE repository that has had this cleanup process performed on it.
+
+---
+
+## **AS OF THE ISLE v.1.4.0 release in November 2019** All ISLE endusers are asked to run this script following the criteria below in the "Do I need to run this script? section."
```

Comment:
"Do I need to run this script?" section.

Reply:
Removed and edited to match clearer more directed README.md